platform/core/src/redux/reducers/preferences.js

```diff
@@ -1,12 +1,6 @@
 import cloneDeep from 'lodash.clonedeep';
 
 const defaultState = {
-  // First tab
-  hotkeyDefinitions: [
-    // commandName, label, keys
-    // [{ zoom: { label: 'Zoom', keys: ['z'] }}]
-  ],
-  // Second tab
```

Comment:
Good call. We should be able to do this with windowLevelData soon as well

Reply:
WindowLevel is also done in another PR already, but the other PR is relying in this PR's changes as well.